opba-protocols/xs2a-protocol-tests/xs2a-bdd-sandbox/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/sandbox/servers/SandboxServers.java

```diff
@@ -55,4 +66,53 @@ public SELF enabled_redirect_sandbox_mode(String aspspProfileUri) {
 
         return self();
     }
+
+    public SELF create_new_user(String tppServerUrl, String login, String password) {
```

Comment:
Please rename `create_new_user` -> `create_new_user_in_sandbox_tpp_management`
Please rename `tppServerUrl` to sandboxTppManagementServerUrl

TPP is actually OpenBanking, so it is confusing

Reply:
renamed